src/desktop/components/fair_layout/test/templates/nav.coffee

```diff
@@ -49,7 +49,7 @@ describe 'Header template', ->
 
     it 'renders the user header', ->
       user = new CurrentUser fabricate 'user'
-      @$template.html().should.containEql user.get('name')
+      @$template.html().should.containEql '/user/saves'
```

Comment:
Good to know that clicking the user icon leads us to the first tab of the usr settings (`/user/saves`)!